apps/prairielearn/src/pages/partials/questionScorePanel.ejs

```diff
@@ -46,6 +46,16 @@
     </tbody>
   </table>
 
+  <% if (assessment.help_link) {%>
+    <div class="card-footer">
+      <div id="linkNav">
```

Comment:
Is this id necessary?

Reply:
It may not be, this is from the files I copied.

Reply:
Short test has been added here https://github.com/PrairieLearn/PrairieLearn/pull/7929/commits/4ea0074909caf29f922328ae4a149b2bb825bb0b